### DISCUSSION TITLE
vertical bar and sizes.liquid

### DISCUSSION THREAD

**Discussion by 0323pin** [leftwm/leftwm#731] [General]
Hi,

Is it possible to have vertical space reserved for a bar? If so, how?
I've just built a fork of lemonbar that allows the bar to be displayed vertically and I'm trying to figure what to change in my `sizes.liquid`, `up` and `template.liquid` to get it working. Right now the bar displays as it should but, clients cover the bar and there's empty space at the top.

Any hints?
Thanks

**Comment by VuiMuich** [comment]
I guess technically it should be possible, that the bar requests a vertically reserved space as well.
I am not sure if I remember this correctly, but I believe I once saw some setup with leftwm+eww with a vertical bar..

If the bar is not able to reserve its space you can try to work around this with [gutters](https://github.com/leftwm/leftwm/wiki/Theme-Config#gutters).

**Reply by 0323pin** [reply]
Thanks, I'll look into gutters. The problem seems to be how this fork works. It actually rotates the bar, which I guess is causing the initially reserved space to be on top, prior to rotation.
